packages/product-download-page/index.tsx

```diff
@@ -113,7 +113,7 @@ export default function ProductDownloadsPage({
     <ProductMetaProvider product={product}>
       <HashiHead title={`Downloads | ${name} by HashiCorp`} />
       <div className={`${styles.root} ${themeClass || ''} ${className || ''}`}>
-        <h1>
+        <h1 className={styles.pageTitle}>
```

Comment:
These headings were relying on global heading styles - the only place in `react-components` where we do this. We're trying to remove those duplicative styles; updating this component means we can isolate global heading styles to `www-next` only.